Rubberduck.Parsing/Preprocessing/VBAPredefinedCompilationConstants.cs

```diff
@@ -18,52 +18,14 @@ public VBAPredefinedCompilationConstants(double vbVersion)
         public const string WIN16_NAME = "Win16";
         public const string MAC_NAME = "Mac";
 
-        public bool VBA7
-        {
-            get
-            {
-                return _vbVersion >= 7;
-            }
-        }
-
-        public bool VBA6
-        {
-            get
-            {
-                return _vbVersion >= 6;
-            }
-        }
-
-        public bool Win64
-        {
-            get
-            {
-                return IntPtr.Size >= 8;
-            }
-        }
-
-        public bool Win32
-        {
-            get
-            {
-                return true;
-            }
-        }
-
-        public bool Win16
-        {
-            get
-            {
-                return false;
-            }
-        }
-
-        public bool Mac
-        {
-            get
-            {
-                return false;
-            }
-        }
+        //For some reason, the predefined compilation arguments in VBA are of type Single
```

Comment:
This says `Single` but we use `short`. VBA's equivalent of `short` would be `Integer`, not `Single`.

Reply:
My bad, will correct.